SingularityUI/app/templates/deployDetail/deployInfo.hbs

```diff
@@ -0,0 +1,60 @@
+<div><div class="page-header">
+    <h2>Info</h2>
+</div>
+
+<div class="row">
+    <ul class="list-unstyled horizontal-description-list">
+        <li class="col-md-4">
+            <div>
+                <h4>Initiated<a data-clipboard-text="{{ timestampFromNow data.deployMarker.timestamp }}">Copy</a></h4>
+                <p>{{ timestampFromNow data.deployMarker.timestamp }}</p>
+            </div>
+        </li>
+        <li class="col-md-4">
+            <div>
+                <h4>Completed<a data-clipboard-text="{{ timestampFromNow data.deployResult.timestamp }}">Copy</a></h4>
+                <p>{{ timestampFromNow data.deployResult.timestamp }}</p>
+            </div>
+        </li>
+        <li class="col-md-4">
+            <div>
+                <h4>Command<a data-clipboard-text="{{ timestampFromNow data.deploy.command }}">Copy</a></h4>
```

Comment:
I think this needs to be data.deploy.executorData.cmd on this line and the one below. Currently coming up blank in UI